cmd/main.go

```diff
@@ -94,7 +94,13 @@ func main() {
 		os.Exit(1)
 	}
 
-	b := controller.NewDpuOperatorConfigReconciler(mgr.GetClient(), mgr.GetScheme(), dpuDaemonImage, vspImages)
+	webhookImage := os.Getenv("WEBHOOK_IMAGE")
+	if webhookImage == "" {
```

Comment:
So if I set "ENABLE_WEBHOOKS to false, I will still get this particular webhook: https://github.com/openshift/dpu-operator/blob/main/cmd/main.go#L114

Should we converge the logic to deploy these?

Maybe we need a more descriptive name in general to differentiate these webhooks

Reply:
You will... but this webhook is specified as a pod called the network-resources-injctor when deployed.

Reply:
I mean from the perspective of someone reading the code